node_package/src/buildConsoleReplay.js

```diff
@@ -0,0 +1,23 @@
+import RenderUtils from './RenderUtils';
+
+export default function buildConsoleReplay() {
+  let consoleReplay = '';
+
+  const history = console.history;
+
+  if (history && history.length > 0) {
+    history.forEach(msg => {
+      const stringifiedList = msg.arguments.map(arg => {
+        try {
+          return (typeof arg === 'string' || arg instanceof String) ? arg : JSON.stringify(arg);
```

Comment:
is this for performance?  Couldn't we just run it through stringify no matter what, or does that not work?